services/portal/GWAS/GWASTasks.js

```diff
@@ -76,8 +76,9 @@ module.exports = {
   },
 
   async selectDifferentType() {
-    I.seeElement(GWASProps.selectDifferentType);
-    I.click(GWASProps.selectDifferentType);
+    I.seeElement(GWASProps.SelectDifferentType);
+    I.click(GWASProps.SelectDifferentType);
+    I.wait(2);
     I.seeElement(GWASProps.AreyousurePopup);
```

Comment:
can we change this to waitForElement and add a wait time of 2 there instead of an explicit wait?

Reply:
Actually have tried with waitForElement but it not worked, only then have used hard wait.